pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -67,11 +67,9 @@ type ActualStateOfWorld interface {
 	SetVolumeMountedByNode(volumeName api.UniqueVolumeName, nodeName string, mounted bool) error
 
 	// ResetNodeStatusUpdateNeeded resets statusUpdateNeeded for the specified
-	// node to false indicating the AttachedVolume field of the Node's Status
-	// object has been updated.
-	// If no node with the name nodeName exists in list of attached nodes for
-	// the specified volume, an error is returned.
-	ResetNodeStatusUpdateNeeded(nodeName string) error
+	// node to true indicating the AttachedVolume field of the Node's Status
+	// object needs to be updated by the node updater again.
+	ResetNodeStatusUpdateNeeded(nodeName string)
```

Comment:
This method can result in a `failed to setNodeStatusUpdateNeeded` error if nodeName does not exist. I like documentation comments to capture 1) what the input is, 2) what the normal output is, and 3) what results in an error. So I would leave the comment as is. That said, I'll leave it up to you to decide.